scripts/identify-web-features.js

```diff
@@ -0,0 +1,320 @@
+    return await response.json();
+  } catch (error) {
+    console.error(`Error fetching the feature data for ${id}:`, error);
+    return null;
+  }
+}
+
+function getBaselineStatusAsMarkdown(feature) {
+  if (feature.status && feature.status.baseline === "high") {
+    return "Widely Available";
+  } else if (feature.status && feature.status.baseline === "low") {
+    return "Newly Available";
+  }
+  return "Limited Availability";
+}
+
+function getDocsAsMarkdown(feature) {
+  if (!feature.mdnUrls.length) {
+    return "";
+  }
+
+  const docs = feature.mdnUrls.map(url => `[${url.title}](${url.url})`).join(", ");
+  return `* **Docs:** ${docs}\n`;
+}
+
+function getStandardPositionsAsMarkdown(feature) {
+  if (!feature.standardPositions.mozilla.url && !feature.standardPositions.webkit.url) {
+    return "";
+  }
+
+  let pos = "* **Standard positions:** ";
+
+  if (feature.standardPositions.mozilla.url) {
+    pos += `[Mozilla](${feature.standardPositions.mozilla.url})`;
+  }
+  if (feature.standardPositions.webkit.url) {
+    pos += (pos ? ", " : "") + `[WebKit](${feature.standardPositions.webkit.url})`;
+  }
+
+  return pos + "\n";
+}
+
+function getUseCounterAsMarkdown(feature) {
+  if (!feature.useCounters.chromeStatusUrl) {
+    return "";
+  }
+  return `* **Chrome use counter:** [chromestatus.com](${feature.useCounters.chromeStatusUrl})\n`;
+}
+
+function getSurveysAsMarkdown(feature) {
+  if (!feature.stateOfSurveys || !feature.stateOfSurveys.length) {
+    return "";
+  }
+
+  const surveys = feature.stateOfSurveys.map(survey => {
+    return `[${survey.name} (${survey.question} question)](${survey.link})`;
+  }).join(", ");
+
+  return `* **State of CSS/JS/HTML surveys:** ${surveys}\n`;
+}
+
+function getPreviousInteropsAsMarkdown(feature) {
+  if (!feature.interop.length) {
+    return "";
+  }
+
+  const interops = feature.interop.map(i => {
+    return `[${i.year}](https://wpt.fyi/interop-2024?feature=${i.label})`;
+  }).join(", ");
+
+  return `* **Included in previous Interop iterations:** ${interops}\n`
+}
+
+function getWPTLinkAsMarkdown(feature) {
+  if (!feature.wptLink) {
+    return "";
+  }
+  return `* **WPT tests:** [wpt.fyi](https://wpt.fyi/results/?q=feature:${feature.id})\n`;
+}
+
+// Generate the markdown content for the given feature.
+function getMarkdownContentForFeature(feature) {
+  let str = `### Feature **${escapeFeatureName(feature)}**\n\n`;
+  str += `* **ID:** ${feature.id}\n`;
+  str += `* **Name:** ${escapeFeatureName(feature)}\n`;
+  str += `* **Description:** ${feature.description_html}\n`;
+  str += `* **Baseline status:** ${getBaselineStatusAsMarkdown(feature)}\n`;
+  str += getDocsAsMarkdown(feature);
+  str += getStandardPositionsAsMarkdown(feature);
+  str += getUseCounterAsMarkdown(feature);
+  str += getSurveysAsMarkdown(feature);
+  str += getPreviousInteropsAsMarkdown(feature);
+  str += getWPTLinkAsMarkdown(feature);
```

Comment:
Do you have a issue where the WPT link is added and working? It seems like everything is there in https://github.com/captainbrosset/interop-proposals-bot/issues/1 for it to be added, but it's not. Maybe there's a bug here?

Reply:
Good catch, fixed it now.